src/vscode/vscode-languageclient-js-stubs/vscode_languageclient.ml

```diff
@@ -189,6 +189,38 @@ module StaticFeature = struct
       [@@js.builder]]
 end
 
+module StreamInfo = struct
+  include Interface.Make ()
+
+  type njs_stream
+
+  let njs_stream_of_string str : njs_stream =
+    Format.ksprintf
+      Js_of_ocaml.Js.Unsafe.eval_string
+      "new joo_global_object.webSocket (`%s`);"
+      str
+
+    let njs_stream_of_js = Obj.magic
+
+    let njs_stream_to_js = Obj.magic
```

Comment:
@Stevendeo Re-reading this, now I'm a bit puzzled as to why unsafe evaluation and `Obj.magic` are needed. Isn't there a way to have access to websockets in a "safer" way?